### DISCUSSION TITLE
Warning to change SMS 2FA to another Authenticate. I'm not using SMS as 2FA🤔

### DISCUSSION THREAD

**Discussion by kavicastelo** [community/community#129189] [Other Feature Feedback, Questions, & Ideas] [answered]
GitHub warns me to change my 2fa method SMS to another authenticate. But I'm not using SMS to authenticate. Already I'm using the Authenticate App to authorize Github. I have 2 GitHub accounts and both accounts show the same warning. Why is this happening?

PS: One of my GitHub accounts has enabled the GPG key too.

![Capture](https://github.com/community/community/assets/85213722/4c9a1868-28a2-4adc-bf55-028cb46be2f5)

**Comment by davevad93** [comment, marked as answer]
Hi @kavicastelo ,

I think it's not explained in a very clear way, I believe they mean that you need to have more than one 2FA method enabled and avoid using SMS authentication, for instance my warning disappeared when I added an additional 2FA method.

**Reply by kavicastelo** [reply]
Thank you, I'll try to add another 2FA method. But the problem is this warning appears on my device only. I asked a couple of my friends who enabled one 2FA method. (same as mine) and they still didn't get this warning.

The important point is this warning pointed out this point,
`Please configure another 2FA method to reduce your risk of permanent account lockout`
This means they directly asked me to change my current 2FA method. (as I understand)

**Reply by davevad93** [reply]
You're welcome @kavicastelo ,

I think it means you must have more than one `2FA` method enabled so don't disable the method you already have, just add one more, for example I have added an authentication option via GitHub mobile app and the warning is gone.

**Reply by kavicastelo** [reply]
Got it @davevad93 . It might be a possible solution and I'll try it😊

**Reply by davevad93** [reply]
It worked for me, now I have two 2FA methods and no warnings, I'm sure it will work for you as well @kavicastelo .

**Reply by yoda-vid** [reply]
I also have both the "Authenticator app" and "Security keys" options enabled. Since "SMS/Text message" doesn't appear to be a recommended option, does this mean that one must install the "GitHub Mobile" app to avoid this warning?

**Reply by davevad93** [reply]
Hey @yoda-vid , you have two different 2FA methods and still get the warning? That's odd, but I can confirm by activating the `GitHub Mobile` option I got rid of it.

**Reply by yoda-vid** [reply]
Yeah mine looks like the screenshot in the OP with 2 different 2FA's. Maybe they really want me to install the app!

**Reply by karelin** [reply]
@yoda-vid I can confirm I have the same issue.
I've tried GitHub mobile in the past, and it is quite inconvenient tool: I'd prefer to explore repos using my browser.

Does anybody knows how to get an official answer from MS guys?

**Comment by LLUMOAI** [comment]
Hi @kavicastelo If you're using an Authenticator App for both accounts and not SMS, the warning to change your 2FA method could be incorrect.

**Reply by kavicastelo** [reply]
Hi @LLUMOAI ,
Yes, it's incorrect. but I don't know why it appears and need to know how it can affect me further. Is there a possibility of lockout my account from GitHub?
